tools/build/configure_php_templates.sh

```diff
@@ -99,6 +99,16 @@ function configure_from_template() {
     echo "Configuring file ${_OUTPUT_FILE} from ${_ARG_INPUT_FILE}"
 
     configure_file "${_ARG_INPUT_FILE}" >"${_OUTPUT_FILE}"
+
```

Comment:
I'm wondering, maybe it will be easier to create file with list of files to skip static checks on?

Reply:
Yes, I added generated files to excluded list in phpcs.xml.dist